po/fr.po

```diff
@@ -24492,7 +24502,6 @@ msgstr "actions souris"
 #~ msgid "exif data"
 #~ msgstr "données EXIF"
 
-#, c-format
```

Comment:
All those c-format removal are suspicious to me. They are there to ensure that the % string format are not causing troubles.

Reply:
OK, I suspect poedit to have removed these lines, for what reason I don't know.
c-format and no-c-format are being restored.